VERSION

```diff
@@ -0,0 +1 @@
+1.1.1
```

Comment:
What does this version refer to?

Reply:
@fossell In the `CMakeLists.txt`, a version number `1.1.1` was hard-wired.  This PR removes the hard-wiring from the `CMakeLists.txt`, and places it in this file.  If this is an inappropriate number, please suggest what is the appropriate version that this application is at.

Reply:
Thanks for clarifying, post is using v8.0.0 for this app.

Reply:
@fossell version number has been update to 8.0.0